SQL_testing.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="bg">
+	<head>
+		<meta charset="UTF-8"/>
+		<title>JAVA_01</title>
+		
+		<style>
+			body
+			{
+				border: 2px solid black;
+				text-align: center;
+			}
+		</style>
+	</head>
+	<body>		
+		<form method="post" action="SQL_testing.php" onsubmit="return validate()">		
+			<label for="title">Име на предмета: </label>
+			<input type="text" id="title" name="title"><br>
+			<label id="invalidTitle"></label><br>
+			
+			<label for="description">Описание: </label>
+			<input type="text" id="description" name="description"><br>
+			<label id="invalidDescription"></label><br>
+			
+			<label for="lecturer">Преподавател: </label>
+			<input type="text" id="lecturer" name="lecturer"><br>
+			<label id="invalidLecturer"></label><br>
```

Comment:
You can put `required` attribute in input tags just to be sure they will be filled by the user and not left empty.